object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,24 @@
+/* exported omit */
+
+/*
+  - Create newObj variable
+  - Create storage variable
+  - Loop through source object
+    - If current prop is equal to first or second key,
+      - Then add it to storage
+    - Else new object at current prop is equal to current key
+  - Return new object
+*/
+
+function omit(source, keys) {
+  var newObj = {};
+  var storage = {};
+  for (var prop in source) {
+    if (prop === keys[0] || prop === keys[1]) {
```

Comment:
What will you do when there are more that two keys though? Try to rework your solution to handle more than just two keys.